Redux patterns/redux+typescript.md

```diff
@@ -1,21 +1,15 @@
-# Redux + Typescript - Writing good code could mean less boilerplate
-# Redux + Typescript - Quality fast coding 
-# Redux + Typescript - Writing good/quality code could actually mean less boilerplate 
-# Redux + Typescript - Writing quality Javascript was never so fast 
-# Redux + Typescript - Writing quality Javascript just became way faster
-# Redux + Typescript + Immer - Writing safe maintainble code just became blazing fast 
 # Redux patterns - Writing safe maintainable code just became blazing fast 
 ## Part 1 - Redux + Typescript 
 
 
```

Comment:
I reverted back to my dash pattern. I value your opinion, but personally the dash pattern feels right. 

"the most important parts of an application" perhaps we should say something like "the most important parts of the client side of an application" more accurate but perhaps the sentence becomes too long.

Reply:
Completely fair for the dash pattern.

For the second one, my preference is to go with the shorter sentence. I think you don't need to be that specific since we are already discussing in the context of Redux.